ts/features/it-wallet/store/actions/itwRpActions.ts

```diff
@@ -12,7 +12,10 @@ export const itwRpInitialization = createAsyncAction(
   "ITW_RP_INITIALIZATION_FAILURE"
 )<
   { authReqUrl: string; clientId: string },
-  { requestObject: RequestObjectConf; entity: RpEntityConfiguration },
+  {
+    requestObject: RequestObjectConf;
+    entity: Awaited<ReturnType<typeof getRelyingPartyEntityConfiguration>>;
```

Comment:
Why not use directly the type `RelyingPartyEntityConfiguration` instead? https://github.com/pagopa/io-react-native-wallet/blob/72a467154f702e49e57376437b6986f802d89f74/src/trust/types.ts#L152

Reply:
I guess it's not exposed anymore

Reply:
solved